infrastructure/k8s/authorization/deployment.yaml

```
@@ -0,0 +1,37 @@
+            - name: SPRING_PROFILES_ACTIVE
+              value: "dev"
+            - name: JAVA_OPTS
+              value: "-Xmx1g -Xms1g"
+          resources:
+            requests:
+              memory: "1.5Gi"
+              cpu: "500m"
+            limits:
+              memory: "2Gi"
+              cpu: "1"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)